MapboxDirections/MBVisualInstructionComponent.swift

```diff
@@ -45,6 +45,20 @@ open class VisualInstructionComponent: NSObject, NSSecureCoding {
      */
     @objc public var abbreviationPriority: Int = NSNotFound
     
+    /**
+     An array indicating which directions you can go from a lane (left, right, or straight).
+     
+     If the value is `[.left", .straight]`, the driver can go left or straight from that lane. This is only set when the `component` is a `lane`.
```

Comment:
> This property is set to `nil` unless the `type` property is set to `VisualInstructionComponentType.lane`.

Reply:
@1ec5 FYI, `LaneIndication` is an array of lane indications. For obj-C compatibility, we shouldn't set this to an optional array.

Reply:
LaneIndication is an OptionSet, not an array. It just happens that Swift uses the same syntax for option sets as it does for arrays. We should say the property is set to the empty set (or 0 in Objective-C) in this case, and we should describe the property as an option set, not an array.

Reply:
Okay. Noted. In which case, we will indicate that this OptionSet is by default an empty set. Hence, we keep this as a type `isActiveLane` as a type `LaneIndication`.